src/dxtbx/nexus/nxmx.py

```diff
@@ -208,15 +208,6 @@ def definition(self) -> str:
 class NXdata(H5Mapping):
     """NXdata describes the plottable data and related dimension scales."""
 
-    def __getitem__(self, key):
-        return self._handle[key]
-
-    def __iter__(self):
-        return iter(self._handle)
-
-    def __len__(self):
-        return len(self._handle)
-
```

Comment:
@rjgildea, am I correct in my assertion that these are redundant?

Reply:
It looks to be the case. I'm not sure why these ended up duplicated here...